sanic/pages/error.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Mapping
+
+import tracerite.html  # type: ignore
+
+from html5tagger import E
+from tracerite import html_traceback, inspector  # type: ignore
+
+from sanic.request import Request
+
+from .base import BasePage
+
+
+# Avoid showing the request in the traceback variable inspectors
+inspector.blacklist_types += (Request,)
+
+ENDUSER_TEXT = """We're sorry, but it looks like something went wrong. Please try refreshing the page or navigating back to the homepage. If the issue persists, our technical team is working to resolve it as soon as possible. We apologize for the inconvenience and appreciate your patience."""  # noqa: E501
+
+
+class ErrorPage(BasePage):
+    STYLE_APPEND = tracerite.html.style
+
+    def __init__(
+        self,
+        title: str,
+        text: str,
+        request: Request,
+        exc: Exception,
+    ) -> None:
+        super().__init__()
+        # Internal server errors come with the text of the exception,
+        # which we don't want to show to the user.
+        # FIXME: Needs to be done in a better way, elsewhere
+        if "Internal Server Error" in title:
+            text = "The application encountered an unexpected error and could not continue."  # noqa: E501
```

Comment:
We don't need this. It is already handled here: https://github.com/sanic-org/sanic/blob/niceback-error-handling/sanic/errorpages.py#L75